plugin-tester-scala/src/test/scala/example/myapp/helloworld/GreeterServiceSpec.scala

```diff
@@ -4,24 +4,26 @@
 
 package example.myapp.helloworld
 
-import scala.concurrent.Await
-
-import org.scalatest.BeforeAndAfterAll
-import scala.concurrent.duration._
-
-import akka.actor.ActorSystem
+import akka.actor.{ActorSystem, ClassicActorSystemProvider}
 import akka.grpc.GrpcClientSettings
 import akka.stream.ActorMaterializer
 import com.typesafe.config.ConfigFactory
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.Span
-
 import example.myapp.helloworld.grpc._
+import org.junit.runner.RunWith
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.should.Matchers
+import org.scalatest.time.Span
 import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatestplus.junit.JUnitRunner
 
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+@RunWith(classOf[JUnitRunner])
 class GreeterSpec extends Matchers with AnyWordSpecLike with BeforeAndAfterAll with ScalaFutures {
-  implicit val patience = PatienceConfig(5.seconds, Span(100, org.scalatest.time.Millis))
+
+  implicit val patience = PatienceConfig(10.seconds, Span(100, org.scalatest.time.Millis))
```

Comment:
wow! Do we really need 10 seconds of patience?

Meh.

Reply:
5 sec sometimes crashed on my laptop, so I decided increase it drastically.